### ISSUE TITLE
Retire the yum repo at http://dl.bintray.com/gocd/gocd-rpm

### ISSUE THREAD

**Issue by sriramnrn** [gocd/gocd#2602]
##### Issue Type

- Bug Report

##### Summary

The yum repo still has 16.1.0 as of the time of filing this bug report. The 16.8.0 release RPMs are available for direct download.

##### Environment

See below

###### Basic environment details

[root@localhost ~]# cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)

###### Additional Environment Details

NA
##### Steps to Reproduce

[root@localhost ~]# cat /etc/yum.repos.d/thoughtworks-go.repo
[gocd]
name            = GoCD YUM Repository
baseurl         = http://dl.bintray.com/gocd/gocd-rpm
enabled         = 1
gpgcheck        = 0
[root@localhost ~]# yum install go-server
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: linux.mirrors.es.net
 * extras: mirrors.greenmountainaccess.net
 * updates: mirror.stjschools.org
Resolving Dependencies
--> Running transaction check
---> Package go-server.noarch 0:16.1.0-2855 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

=============================================================================================================================
 Package                        Arch                        Version                          Repository                 Size
=============================================================================================================================
Installing:
 go-server                      noarch                      16.1.0-2855                      gocd                      134 M

Transaction Summary
=============================================================================================================================
Install  1 Package

Total download size: 134 M
Installed size: 134 M
Is this ok [y/d/N]: N
Exiting on user command
Your transaction was saved, rerun it with:
 yum load-transaction /tmp/yum_save_tx.2016-08-27.02-06.azX4YX.yumtx

##### Expected Results
We should see 16.8.0 available for download.

##### Actual Results
We see 16.1.0 available for download.

##### Possible Fix
It may be worthwhile setting up a "release" pipeline that updates the website as well as the yum repo over at binary.

##### Log snippets
NA
##### Code snippets/Screenshots
NA

##### Any other info
NA

**Comment by sriramnrn** [comment]
After submitting this report, I noticed that I have gig check turned off. I decided to compare my repo settings with what is on the GoCD docs. There are differences there, notably the download URL itself, and GPG check turned on.

After using the revised repo as advised here: https://docs.go.cd/current/installation/install/server/linux.html#rpm-based-distributions-ie-redhatcentosfedora,

I now see the 16.8.0 version being downloaded and installed.

So perhaps, the corrective action is to not host a yum repo at the older URL at all.
